books/thirdparty/livelibru.py

```diff
@@ -30,6 +31,22 @@
 SEARCH_PAGE_SIZE = 100
 
 
+@dataclasses.dataclass(frozen=True)
+class Book:
+    name: str
+    author_name: str
+    cover_image: str
+    url: str
+    reviews: int
+
+
+class DataclassJSONEncoder(json.JSONEncoder):
```

Comment:
oh, I thought it would encode data calsses by default. That's kinda annoying. 
No action needed right now.